FilterViewController.m

```diff
@@ -26,6 +26,13 @@ - (void)viewDidLoad {
     self.currentSubViewControllerIndex = 0;
 }
 
+- (void)viewDidDisappear:(BOOL)animated{
+    
+    [super viewDidDisappear:animated];
+    
+    [[NSNotificationCenter defaultCenter] removeObserver:self name:UIDeviceOrientationDidChangeNotification object:nil];
```

Comment:
I added the observer just in case the user rotated the device while the "half modal" was up. What I didn't realize was the half modal does not work well on the iPhone. Ryan put it on the tester phone and the keyboard was blocking the piece with text entirely.

I am removing the code for the half modals and going with real modals (the tableviewcontroller modal is a true one). Since view controllers handle rotation well this will no longer be needed.